src/shell/shell.cpp

```diff
@@ -340,17 +340,18 @@ void DOS_Shell::Run(void) {
 	}
```

Comment:
This diff here looks really messy, but it just moves the indent level.
I was reading about different diff methods, and perhaps the "patience" method would make this much more readable.. however, I don't think we can control that in GitHub.